imgix/contentful
# [PE-2942] feat: add filename overlay  to assets

## Description
This PR adds a filename overlay to the `<ImageGallery />`'s `<GridImage />` components.



# Before
`<GridImage />` component did not display the filename overlayed over the last 25% of the image container

# After
`<GridImage />` component displays the filename overlayed over the last 25% of the image container



# Steps to test
- pull down this branch
- run `npm run start`
- open contentful staging env
- go to content
- open content with imgix plugin
- click add image
- see filename overlayed over assets

# 🖼️  Screenshots
> filenames before
<img width="202" alt="filename-before" src="https://user-images.githubusercontent.com/16711614/198704182-312ddf23-0f45-4b5a-9719-a6e061134bfc.png">

> filenames after
<img width="187" alt="asset-filenames-example" src="https://user-images.githubusercontent.com/16711614/198696537-35c46183-5d82-4102-8fc2-5b591c4d29c5.png">

https://user-images.githubusercontent.com/16711614/198696553-3689cb36-cfc2-4968-ab93-5a09a01ebbfc.mov



## Checklist
